src/main/plugins/org.dita.eclipsehelp/build_dita2eclipsehelp_template.xml

```diff
@@ -26,14 +29,21 @@ See the accompanying LICENSE file for applicable license.
 	
     <target name="dita2eclipsehelp"
             unless="noMap"
-        depends="build-init, dita.eclipsehelp.init, preprocess, 
+        depends="dita.eclipsehelp.init, build-init, preprocess, 
                  xhtml.topics,
                  copy-css">
       <antcall target="dita.map.eclipse"></antcall>
+      <jar destfile="${output.dir}/${args.eclipsehelp.jar.name}" basedir="${dita.output.dir}" if:set="args.eclipsehelp.jar.name">
```

Comment:
Done, and made the `<jar>` action into a target that runs after `dita.map.eclipse`.